src/shortcuts/deleteEmptyThoughtOrOutdent.tsx

```diff
@@ -71,11 +71,17 @@ const canExecuteOutdent = (state: State) => {
   if (!cursor || !selection) return false
 
   const offset = selection.focusOffset
+  const focusedEditingValue = state.editingValue || ''
+  const contextOfCursor = pathToContext(cursor)
+  const parentOfCursor = parentOf(contextOfCursor)
+  const filteredList = contextOfCursor.filter(el => !parentOfCursor.includes(el))
```

Comment:
This `filteredList` doesn't make sense so me. Paths and contexts are contiguous, e.g. a → b → c. Filtering it breaks the invariance of contiguity. Maybe you can explain you intention here and I can help point you in the right direction.